addons/sale/models/sale_order_line.py

```diff
@@ -257,6 +257,12 @@ class SaleOrderLine(models.Model):
         compute='_compute_product_uom_readonly',
         help="UX field to know when the UoM can be modified")
 
+    @property
+    def FIELDS_IMPACTING_PRODUCT_PRICE(self):
```

Comment:
What about `PRICE_COMPUTING_FIELDS`? IMO the name does not matter too much, a nice comment describing what it will be used for is more useful.